src/screens/WebView/HelpCenter.js

```diff
@@ -1,39 +1,39 @@
 import * as React from 'react';
-import { ActivityIndicator, SafeAreaView, StyleSheet, View } from 'react-native';
-import { WebView } from 'react-native-webview';
+import {ActivityIndicator, SafeAreaView, StyleSheet, View} from 'react-native';
+import {WebView} from 'react-native-webview';
 
 import Header from '../../components/Header';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 
 const HelpCenter = ({navigation}) => (
-    <View style={styles.container}>
-      <SafeAreaView>
-      <Header title="Help Center" onPress={() => navigation.goBack()}  />
-      </SafeAreaView>
-      <WebView
-        androidHardwareAccelerationDisabled={false}
-        source={{ uri: 'https://bettersocial.org/help' }}
-        showsVerticalScrollIndicator={false}
-        startInLoadingState={true}
-        style={styles.webview}
-        renderLoading={() => (
-          <View style={styles.loading}>
-            <ActivityIndicator size="large" color="#00ADB5" />
-          </View>
-        )}
-      />
-    </View>
-  );
+  <View style={styles.container}>
+    <SafeAreaView>
+      <Header title="Help Center" onPress={() => navigation.goBack()} />
+    </SafeAreaView>
+    <WebView
+      androidHardwareAccelerationDisabled={false}
+      source={{uri: 'https://bettersocial.org/help'}}
+      showsVerticalScrollIndicator={false}
+      startInLoadingState={true}
+      style={styles.webview}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `WebView` component is using the `startInLoadingState` prop set to `true`, which means it will start in a loading state. This is fine if you want to display a custom loading indicator, but it can lead to unnecessary rendering if the WebView loads quickly. If the loading of the WebView isn't usually a problem, consider removing this prop for performance reasons.



<!-- This is an auto-generated comment by OSS CodeRabbit -->